extensions/amp-access-scroll/0.1/scroll-impl.js

```diff
@@ -159,6 +160,13 @@ export class ScrollAccessVendor extends AccessClientAdapter {
               new ScrollElement(this.ampdoc).handleScrollUser(
                   this.accessSource_, config);
               addAnalytics(this.ampdoc, config);
+              if (response.features && response.features.readDepth) {
```

Comment:
A heads up that the usage here will be broken after AMP migrate to singlepass compilation. 
The object field name will be mangled here, for example `response.features` to `response.a`. The response object is actually a JsonObject. If you want to preserve the field name, you want to migrate to `response['features']` instead. 
This isn't an issue right now, but will be soon. Please create a seprate PR to fix the usage here. cc @erwinmombay